app/controllers/image_uploads_controller.rb

```diff
@@ -34,15 +34,7 @@ def create
       return
     end
 
-    cloudinary_link(uploaders)
-  end
-
-  def cloudinary_link(uploaders)
-    links = if params[:wrap_cloudinary]
-              [ApplicationController.helpers.cloud_cover_url(uploaders[0].url)]
-            else
-              uploaders.map(&:url)
-            end
```

Comment:
There's no need to pre-wrap these uploaded image URLs with Cloudinary/imgproxy. Instead, we should store it as a raw image URL which provides greater flexibility on how we want to optimize it later.  Plus, I searched through the codebase, and everywhere I looked, we are already trying to optimize it again, ie `cloud_cover_url(article.main_image)`.